docs/content/users/quickstart.md

```diff
@@ -341,7 +341,76 @@ Here's a quickstart instructions for a number of different environments:
     
     This way we only need to change the value of `DB_CONNECTION` to `ddev` in the `.env` to work with the `db` service.
     This is very handy if you have a local database installed and you want to switch between the connections faster by changing only one variable in `.env`
+    
+    ### Using Vite
+    The default Laravel Vite template will not work with DDEV. To make DDEV and Vite play nicely a few additional steps are required.
```

Comment:
It might be nice to reword for brevity and to orient the reader to the steps that follow:

```suggestion
    The default Laravel Vite template doesn’t work with DDEV, so you’ll need to set up a Vite server and configure Laravel to use it.
```